libs/domain/checkout/guest/guest.component.ts

```diff
@@ -0,0 +1,63 @@
+import { CheckoutMixin } from '@spryker-oryx/checkout';
+import { resolve } from '@spryker-oryx/di';
+import { defaultOptions } from '@spryker-oryx/experience';
+import { RouterService } from '@spryker-oryx/router';
+import { SemanticLinkService, SemanticLinkType } from '@spryker-oryx/site';
+import { Size } from '@spryker-oryx/ui';
+import { hydratable, i18n, signal } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { styles } from './guest.styles';
+
+@defaultOptions({
+  enableGuestCheckoutByDefault: true,
+})
+@hydratable('window:load')
+export class CheckoutGuestComponent extends CheckoutMixin(LitElement) {
+  static styles = [styles];
+
+  protected linkService = resolve(SemanticLinkService);
+
+  protected checkout = signal(
+    this.linkService.get({ type: SemanticLinkType.Checkout })
+  );
+
+  protected checkoutLogin = signal(
+    this.linkService.get({ type: SemanticLinkType.CheckoutLogin })
+  );
+
+  connectedCallback(): void {
```

Comment:
Can cause uncontrolled behaviour when called earlier then signals are resolved. I believe Stan suggested to use `effect` instead

Reply:
I replied to that suggestion (and we had effect before, but it's not working in this case)